README.md

```diff
@@ -30,6 +30,16 @@ You can find a detailed explanations on the [Prebid Universal Creative](http://p
 </script>
 ```
 
+Creative created like described above will work for all formats:
+- amp
+- banner
+- mobile
+- native
```

Comment:
From some initial tests I was running with a new GAM creative using the above code template, I'm not sure if simply writing 'native' here is accurate.  From what I can tell, it seems like not all native setups that we support will be handled with just the code above. 

Specifically the scenario where the native template is set within the creative's adserver.  In that case, the shell of the creative's HTML is included above the script tag that loads the PUC native script (whether it's the new `native.js` or the previous incarnations of `native-render.js` or `native-trk.js`).
https://docs.prebid.org/prebid/native-implementation.html#native-template-in-adserver

If that's indeed the case, then the text here should highlight the restrictions (or give a link to the prebid.org native docs that can better explain what scenarios have specific requirements).

Reply:
I've fixed the [documentation PR](https://github.com/prebid/prebid.github.io/pull/3645/commits/24b5e38ba896d3fda37e2acd632dad57e6f48538) to specify the `document` object as a first argument of the `renderAd` function. 

In general, I don't really like the first scenario of the native approach, where the template is specified in GAM; it looks to me as something that's being carried on for compatibility with the past. It's also the scenario that is worse to customize because it requires to access GAM at every change. Anyway, the PR is compatible with that scenario too; i just tested it. Thanks for catching the mismatch.